### PR TITLE
gcompris 1.1

### PR DESCRIPTION
**Pull request by cho2**

### CODE REVIEW

File: org.kde.gcompris.appdata.xml [hunk 1]
```diff
@@ -808,6 +808,6 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
-    <release version="1.0" date="2020-11-18"/>
+    <release version="1.1" date="2020-03-21"/>
```
Comment (petitlapin) [inline org.kde.gcompris.appdata.xml:811]: 2021?

**Review by petitlapin — COMMENTED** [review]
Thank you for the PR

File: org.kde.gcompris.appdata.xml (hunk 1 above)
Comment (cho2) [reply org.kde.gcompris.appdata.xml]: My bad, sorry for this. Already fixed.

**Comment by eszlari** [comment]
` org.kde.gcompris.appdata.xml` can be deleted, the one in the tarball is valid now

**Pull request merged**